src/main/java/domain/ResultCommand.java

```diff
@@ -0,0 +1,27 @@
+package domain;
+
+import java.util.List;
+
+public class ResultCommand {
+
+    private static final String INVALID_COMMAND_ERROR_MESSAGE = "all 이나 Player 의 이름만 입력이 가능합니다.";
+
+    private final Players players;
+
+    public ResultCommand(Players players) {
+        this.players = players;
+    }
+
+    public List<Player> getCommandResult(Command command) {
+        if (command.isAllCommand()) {
+            return players.getPlayers();
+        }
+
+        return List.of(players.getPlayers()
+                .stream()
+                .filter(command::isCommandMatches)
+                .findAny()
+                .orElseThrow(() -> new IllegalArgumentException(INVALID_COMMAND_ERROR_MESSAGE)));
+    }
```

Comment:
```suggestion
        return List.of(players.findBy(command));
    }
```
혹은
```suggestion
        return List.of(players.findByName(command.getCommand()));

    }
```
요런 방법도 고민해보면 좋을것 같아요

Reply:
이런게 정말 객체에게 메세지를 보내는거군요..